Automation/automation_in_python/menu.py

```diff
@@ -0,0 +1,47 @@
+
+import subprocess
+
+file = open("../banner.txt","r")
+for line in file:
+    print(line)
+file.close()
+
+def menu():       #the menu. 8 options
+    print("\n1. Update checker. ")
+    print("2. Search for Linux. ")
+    print("3. Search for Linux-TW. ")
+    print("4. Search for Windows. ")
+    print("5. Search for OSX. ")
+    print("6. Search for all types of workers. ")
+    print("7. Edit the script. ")
+    print("8. Exit.\n")
+
+menu()
+
+while True:     #the 'switch:case' method to choose an option
+    option = int(input("Choose an option: "))
+    if option == 1:
```

Comment:
Please implement switch-case scenario instead of elif's.
Example:https://jaxenter.com/implement-switch-case-statement-python-138315.html